partcad/src/partcad/assembly_factory_alias.py

```
@@ -31,31 +29,28 @@
                 self.source_assembly_name = config["source"]
             else:
                 self.source_assembly_name = config["name"]
-                if not "project" in config:
-                    raise Exception(
-                        "Alias needs either the source part name or the source project name"
-                    )
+                if "project" not in config and "package" not in config:
+                    raise Exception("Alias needs either the source assembly name or the source project name")
```

Ruff (0.8.2)

Create your own exception

(TRY002)